Cockpit 235 (#372)
cockpit-project/cockpit-project.github.io: _posts/2021-01-07-cockpit-235.md

---
title: Cockpit 235
author: pitti
date: 2021-01-07
tags: cockpit linux
slug: cockpit-235
category: release
summary: Cockpit with SSH key improvements and motd editing
---

Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).  We release regularly.

Here are the release notes from Cockpit version 235.

> ## Login: Better handling of SSH host keys

Reviewer: ```suggestion
## Login: Improved handling of SSH host keys
```

Author: Applied, thank you!

> When connecting to a secondary host directly from the login page, unknown and changed host keys are handled in a better way. The situation is explained and accepted keys are remembered in the Browser:

Reviewer: ```suggestion
This release improves key management while logging into secondary hosts. The log in screen now explains unknown and altered keys&mdash;and also how to verify these new keys. Browsers remember accepted keys in client-side storage to improve the user experience.
```

1. Rearranged a lot of text to make it active voice and simplify the text.
2. "Browser" is not a proper noun, so it should be "browser". However, I rewrote it and started the sentence with "browser", so it's capital anyway in my rewrite… but for a different reason. :wink:
3. Clarified that this is in browser storage.
4. There shouldn't be a colon here.
5. Used an [em dash](https://www.lexico.com/grammar/dash-and-em-dash) (`&mdash;` in HTML (and Markdown), which looks like —) to connect a related sentence fragment to the previous sentence. I did this to offset the "and", so it doesn't look like an ambiguous "x and y and x". Without this, it _could_ look like "x and (y and z)" instead of "(x and y) and z".

![login page SSH fingerprint](/images/login-fingerprint.png)

## Overview: Editable motd

> The displayed motd now has an edit button where you can change `etc/motd`:

Reviewer: ```suggestion
The Overview page's motd now has an edit button to change `/etc/motd`.

*[motd]: message of the day
```

1. I removed the colon here too.
2. Specified Overview page.
3. Dropped "where you can".
4. I _think_ I'm using the footnote correctly here. It should show up as a dotted underline which defines the term both in screen readers and on also on hovers in browser.
